multinet/api.py

```diff
@@ -175,6 +184,7 @@ def delete_workspace(workspace: str) -> Any:
 
 @bp.route("/workspaces/<workspace>/name", methods=["PUT"])
 @use_kwargs({"name": fields.Str()})
+@require_maintainer
```

Comment:
`write` lets you upload and delete tables. Renaming an entire workspace seems like it falls under maintainer/owner duties, rather than people authorized to upload data.

I didn't mention this in the RFC because I forgot we had a renaming endpoint. I will update that RFC with a PR, but I disagree that anyone who can upload data should also be able to rename the whole workspace.

Reply:
My bad, I had misread this endpoint as renaming tables/something else, instead of workspaces. I agree with what you said.